optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/tenant/Tenant.java

```diff
@@ -66,12 +65,12 @@
 
     @SuppressWarnings("unused")
     public Tenant() {
-        this.configuration = new TenantConfiguration(this, 1, DayOfWeek.MONDAY);
+        this.configuration = new TenantConfiguration(0, 1, DayOfWeek.MONDAY,100,10);
     }
 
     public Tenant(String name) {
         this.name = name;
-        this.configuration = new TenantConfiguration(this, 1, DayOfWeek.MONDAY);
+        this.configuration = new TenantConfiguration(0, 1, DayOfWeek.MONDAY,100,10);
```

Comment:
Duplication of the defaults between line 73 and 68.
Suggestion, move the defaults into the TenantConfiguration class (where we except to find them more than here),
by simply setting the fields as such.

Reply:
ah looks like there are already there too, in the constructor (which is also fine). Remove them here anyway.